client/src/components/ManageBookings/BookingCard/BookingCard.tsx

```diff
@@ -1,32 +1,44 @@
 import { Typography, Box } from '@mui/material';
 import SettingsIcon from '@mui/icons-material/Settings';
 import useStyles from './useStyles';
+import { createImageFromInitials } from '../../../helpers/makeAnImageFromName';
+import { props } from '../../../interface/manageBooking';
+import { convertDate } from '../../../helpers/APICalls/convertDateRightFormat';
+
+const BookingCard = ({ name, start, status, photo }: props): JSX.Element => {
+  const letterImage = createImageFromInitials(name);
```

Comment:
https://v4.mui.com/components/avatars/#letter-avatars 

Prefer to use MUI for the letter avatar.